Home_Task_7.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Task 7: String conversion.
+
+    Description:
+
+Convert a non-negative integer num to its English words representation.
+Example 1:
+Input: num = 123
+Output: "One Hundred Twenty Three"
+
+let's take that number always > 100 and only three digits
+100 <= num <= 999
+"""
+
+ones = {'1': 'One', '2': 'Two', '3': 'Three', '4': 'Four', '5': 'Five',
+        '6': 'Six', '7': 'Seven', '8': 'Eight', '9': 'Nine'}
+dozens = {'2': 'Twenty ', '3': 'Thirty ', '4': 'Forty ', '5': 'Fifty ',
+          '6': 'Sixty ', '7': 'Seventy ', '8': 'Eighty ', '9': 'Ninety '}
+hundreds = {'1': 'One Hundred ', '2': 'Two Hundreds ', '3': 'Three Hundreds ',
+            '4': 'Four Hundreds ', '5': 'Five Hundreds ', '6': 'Six Hundreds ',
+            '7': 'Seven Hundreds ', '8': 'Eight Hundreds ',
+            '9': 'Nine Hundreds '}
+teens = {'1': 'eleven', '2': 'Twelve', '3': 'Thirteen', '4': 'Fourteen',
```

Comment:
the cognitive dissonance  when 1 -> means 11 ).
Better here to store values as 11 -> 'Eleven', 12 - > 'Twelve' and within code check 
if num_str[1] == '1'  and then take two numbers together   as str (num_str[1] + num_str[2]) and search corresponding value in dictionary. As minimum code will have no  the cognitive dissonance blocks, will be readable and clear